packages/@aws-cdk/aws-ec2-alpha/lib/route.ts

```diff
@@ -175,6 +175,40 @@ export interface NatGatewayProps extends NatGatewayOptions {
   readonly vpc?: IVpcV2;
 }
 
+/**
+ * Options to define a VPC peering connection.
+ */
+export interface VPCPeeringConnectionOptions {
+  /**
+   * The VPC that is accepting the peering connection.
+   */
+  readonly acceptorVpc: IVpcV2;
+
+  /**
+   * The AWS account ID of the acceptor VPC owner.
```

Comment:
I think this comment is not correct

Reply:
good catch! [e810bd7](https://github.com/aws/aws-cdk/pull/31645/commits/e810bd75bbd8d11fc480384760ffb0d19085bbfa)